src/CBot/CBotClass.h

```diff
@@ -303,6 +303,12 @@ class CBotClass
      */
     static void DefineClasses(std::list<CBotClass*> pClassList, CBotCStack* pStack);
 
+    /*!
+     * \brief Get the list of user-defined methods in this class.
+     * \return List of methods, can be empty.
+     */
+    const std::list<CBotFunction*>& GetFunctions();
```

Comment:
```suggestion
    const std::list<CBotFunction*>& GetFunctions() const;
```

Reply:
This new 'suggest changes' feature is nice, I didn't even know it existed.